example/form/CAP_Adrenal.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <FormDesign xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema" ID="129.1000043" baseURI="www.cap.org/eCC/SDC/IHE" xmlns="http://healthIT.gov/sdc">
   <Header styleClass="left" ID="Header.129.1000043" title="ADRENAL GLAND">
-    <OtherText name="Copyright" type="meta" styleClass="copyright" val="(c) 2016 College of American Pathologists.  All rights reserved.  License required for use." />

```

Comment:
Removed Copyright OtherText element due to non-uniqueness vs. the other such element in the Footer element (the name attribute enforces uniqueness).